### PR TITLE
Removed registration process type option

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | This option is not useful anymore. |
| Type? | improvement |
| Category? | BO |
| BC breaks? | yes |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1047 |
| How to test? | You can't see the option displayed anymore, the code wasn't used at this point so the front office was already aware of this deprecation. |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

**Comment by vincentbz** [comment]
Tested and ok, thanks !

File: controllers/admin/AdminCustomerPreferencesController.php [hunk 1]
```diff
@@ -27,7 +27,8 @@
 /**
  * @property Configuration $object
  */
-class AdminCustomerPreferencesControllerCore extends AdminController
+class
+AdminCustomerPreferencesControllerCore extends AdminController
```
Comment (maximebiloe) [inline controllers/admin/AdminCustomerPreferencesController.php]: Why the new line?

Comment (mickaelandrieu) [inline controllers/admin/AdminCustomerPreferencesController.php]: fixed, thank you

**Comment by vincentbz** [comment]
status: 'PM approved'

**Comment by vincentbz** [comment]
status: 'QA approved'

**Comment by maximebiloe** [comment]
Thank you!

**Pull request merged**